### PR TITLE
Ci check shell

### PR DESCRIPTION
**Pull request by kaikai1024**
### Description of the Change

`shellcheck scripts/*.sh env.sh`

*Not a must-have, put it into travis*

### Applicable Issues

#412

### CODE REVIEW

**Comment by kaikai1024** [comment]
I'll check it later. I's weird, it's ok on my own environment.

**Comment by leeyr338** [comment]
这个 PR 好奇怪，我多次尝试直接进入 ci 的环境下执行都是正确的。但是 rerun 还是错误。

**Comment by kaikai1024** [comment]
> 这个 PR 好奇怪，我多次尝试直接进入 ci 的环境下执行都是正确的。但是 rerun 还是错误。

Would you help to run the failed tests on your own env?
Maybe it's the problem of CI. and i will check the diff file to ensure is there's some random situation.

File: scripts/cita.sh [hunk 1]
```diff
@@ -234,37 +232,35 @@ do_stop() {
...
-        pidfile="${NODE_PATH}/.cita-${service}.pid"
-        if [ ! -e $pidfile ]; then
-            PING_STATUS="pang"
-            return
-        fi
+    pidfile="${NODE_PATH}/.cita-forever.pid"
+    if [[ ! -e "$pidfile" ]]; then
+        PING_STATUS="pang"
+        return
+    fi
```
Comment (ouwenkg) [inline scripts/cita.sh]: 这里不太对吧, 之前是会查询各个 service 的, 改了之后, 只查询 forever 吗

File: scripts/release.sh [hunk 2]
```diff
@@ -42,7 +42,7 @@ done
 #strip                                     target/install/bin/*
 
 # 2) cita
-cp -rf scripts/cita                        target/install/bin/
+cp -rf scripts/cita.sh                      target/install/bin/cita
```
Comment (ouwenkg) [inline scripts/release.sh]: `cp -rf scripts/cita.sh                      target/install/bin/cita.sh` ?

File: scripts/cita.sh (hunk 1 above)
Comment (kaikai1024) [reply scripts/cita.sh]: 之前也只是查询 forever， 那个 for 循环里面只有 forever，没用的代码

File: scripts/release.sh (hunk 2 above)
Comment (kaikai1024) [reply scripts/release.sh]: Np.
Rename `cita` to `cita.sh` for using shell-check.
But still use `bin/cita usage`.

Comment (ouwenkg) [reply scripts/release.sh]: OK, I know

**Pull request closed without merging**